cocoa/neural/models.py

```diff
@@ -81,7 +81,8 @@ class MeanEncoder(EncoderBase):
     """
     def __init__(self, num_layers, embeddings, embed_type='utterance'):
         super(MeanEncoder, self).__init__()
-        self.num_layers = num_layers
+        n_layers = 1 if embed_type == "scenario" else num_layers
+        self.num_layers = n_layers
```

Comment:
num_layers should be specified in cmd line. should not be changed here in an ad-hoc way.

Reply:
num_layers is specified by command line, and i change it ad-hoc for scenario which expects (1 x batch_size x hidden_dim), rather than (num_layers x batch_size x hidden_dim).  i agree this is messy.  how would you handle?  I guess that depends on how we end up wiring together the model.forward() function?